pkgs/by-name/ls/lsfg-vk-ui/package.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  glib,
+  pango,
+  gdk-pixbuf,
+  gtk4,
+  libadwaita,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "lsfg-vk-ui";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "PancakeTAS";
+    repo = "lsfg-vk";
+    tag = "v${version}";
+    hash = "sha256-nIyVOil/gHC+5a+sH3vMlcqVhixjJaGWqXbyoh2Nqyw=";
+  };
+
+  cargoHash = "sha256-hIQRS/egIDU5Vu/1KWHtpt4S26h+9GadVr+lBAG2LDg=";
```

Comment:
Just a note:

Normally I'd ask for `src` & `version` to be inherited from the main `lsfg-vk` package.

But provided that this one needs `cargoHash` and the "main" package doesn't it's better to have a separate `src` definition to ensure that an auto updater doesn't just update the "main" package leaving `cargoHash` in the ui package out of sync.